recipe/conda_build_config.yaml

```diff
@@ -543,6 +543,9 @@ python:
   - 2.7
   - 3.6
   - 3.7
+  # pypy only supports win32 so far
+  - pypy.3.6.7.1  # [unix or win32]
+  - pypy.2.7.7.1  # [unix or win32]
```

Comment:
Do we have win32 builds?

Reply:
PyPy has only win32, no win64 at this time